validatorPassword.js

```diff
@@ -0,0 +1,68 @@
+// Must be a minimum of 8 characters
+function validatePassword (password) {
```

Comment:
The test file is looking for index.js and instead you named it validatorPassword.js so it has zero idea where to find the information it is trying to test